src/docs/index.md

```diff
@@ -128,6 +128,96 @@ title: Akka.NET Documentation
 - [Scheduler](Scheduler)
 - [Circuit Breaker](CircuitBreaker)
 
+### Akka Streams (work in progress)
+*we are currently in the progress of porting the streams documentation for Akka.Net. As such not all sections will be available*
+
+- [Introduction](streams/introduction)
+	- [Motivation](streams/introduction#motivation)
+	- [How to read these docs](streams/introduction#how-to-read-these-docs)
+- [Quick Start Guide](streams/quickstart)
+	- [Reusable Piece](streams/quickstart#reusable-pieces)
+	- [Time-Based Processing](streams/quickstart#time-based-processing)
+- [Reactive Tweets](streams/reactivetweets)
```

Comment:
Me either. Once ill get to that section ill investigate. If i cant find anything suitable. Ill skip it the whole section.

If by the end it means we wont have a good end-to-end example of the streams API. We will have to build our own sample imho.